src/service/video/docking.js

```diff
@@ -561,6 +566,20 @@ export class VideoDocking {
     return element;
   }
 
+  /** @private */
+  hideControlsOnTapOutside_() {
+    listen(this.ampdoc_.getRootNode(), 'mouseup', e => {
```

Comment:
any reason this is not `mousedown`? Imagining user starting to scroll but not lifting their finger, think controsl should go away in that case too.

Reply:
No reason, done!